src/Tizen.Wearable.CircularUI.Forms.Renderer/Shell/NavigationView.cs

```diff
@@ -2,22 +2,75 @@
 using ElmSharp.Wearable;
 using System;
 using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Runtime.CompilerServices;
 using Xamarin.Forms;
 using Xamarin.Forms.Platform.Tizen;
-using ELayout = ElmSharp.Layout;
 using EColor = ElmSharp.Color;
+using ELayout = ElmSharp.Layout;
 
 namespace Tizen.Wearable.CircularUI.Forms.Renderer
 {
     public class NavigationView : ELayout
     {
         readonly int _dafaultIconSize = 60;
 
-        class Item
+        class Item : INotifyPropertyChanged
         {
-            public Element Source { get; set; } 
+            Element _source;
+            public Element Source
+            {
+                get
+                {
+                    return _source;
+                }
+                set
+                {
+                    if (_source != null)
```

Comment:
Don't we need to check if `value` is equal to `_source`?

Reply:
Yes, but it never occurred and no issue even though update with equal value, 
So I think it is unnecessary guard
``` c#
set
{
    if (_source == value) return;
    if (_source != null)
    {
        _source.PropertyChanged -= OnElementPropertyChanged;
    }
    ...
}
```

Reply:
Well, I was just worried that unnecessary actions would happen. But, it's ok if we guarantee that the source doesn't set to the same value. 

At least for now, it seems that there is only one place (`Build()`) where source is set. And it doesn't seem to happen as you said because it already checks and guarantees the changes through `IsUpdate()`.